models.py

```diff
@@ -24,8 +26,6 @@ class User(Base):
 
     Base.metadata.create_all(engine)
 
-    Session = sessionmaker(bind=engine)
-    session = Session()
 
     def __repr__(self):
```

Comment:
Why __repr__ instead __str__?

Reply:
Don't understand what you mean)

Reply:
What's the difference between repr and str?